config/config.go

```diff
@@ -61,6 +61,13 @@ const (
 	DashboardsDiscoveryAuto    = "auto"
 )
 
+// FeatureName is the enum type used for named features that can be disabled via KialiFeatureFlags.DisabledFeatures
+type FeatureName string
+
+const (
+	FeatureLogView FeatureName = "log-view"
```

Comment:
I'd suggest to use "logs-tab" instead of "log-view".
That would help to align better with the UI:
![image](https://user-images.githubusercontent.com/1662329/160582492-676c0964-408d-450c-81df-810f5c2e82df.png)